Gemfile

```diff
@@ -8,3 +8,6 @@ when 'HEAD'
 else
   gem 'grape', version
 end
+
+gem 'grape-swagger-entity', github: 'bugagazavr/grape-swagger-entity'
```

Comment:
I think the ENV variable should be `MODEL_PARSER` to be consistent, should be set to either `grape-swagger-entity` or `grape-swagger-representable` and be `gem ENV['MODEL_PARSER'] if ENV.key?('MODEL_PARSER')`. The specs should pass without a model parser library, I don't think we should have a default and `.travis.yml` would have two entries to make things very explicit.